skore/tests/unit/sklearn/estimator/feature_importance/conftest.py

```diff
@@ -1,12 +1,36 @@
+import numpy as np
 import pytest
-from sklearn.datasets import make_classification, make_regression
+from sklearn.datasets import make_blobs, make_classification, make_regression
 
 
 @pytest.fixture
 def regression_data():
     return make_regression(n_features=5, random_state=42)
 
 
+@pytest.fixture
+def positive_regression_data():
+    X, y = make_regression(n_features=5, random_state=42)
+    return X, np.abs(y) + 0.1
+
+
+@pytest.fixture
+def multi_regression_data():
+    return make_regression(n_features=5, n_targets=2, random_state=42)
+
+
 @pytest.fixture
 def classification_data():
     return make_classification(n_features=5, random_state=42)
+
+
+@pytest.fixture
+def outlier_data():
+    X = np.random.RandomState(42).normal(0, 1, (300, 2))
```

Comment:
Let's use the new generator from numpy.

```suggestion
    rng = np.random.default_rng(42)
    X = rng.normal(0, 1, (300, 2))
```